app/views/servers/show.html.erb

```diff
@@ -214,7 +214,9 @@
               <%= Server.human_attribute_name(:photo) %>
             <% end %>
 
-            <%= image_tag @server.photo.variant(geometry: "x220"), class: "server_photo ms-0 mb-3" %>
```

Comment:
Does `.server_photo` present on css files and still used?

Otherwise could be nice to remove it.

Reply:
Nope that's why I removed it, it's not used anywhere

Reply:
And it has already been removed from any css files?

Reply:
It looks like so. I couldn't find any occurence of `server_photo` on all the project files